dev_requirements.txt

```diff
@@ -1,5 +1,6 @@
 sqlalchemy>=1.1.9
-google-cloud-bigquery>=1.6.0
+google-cloud-bigquery[bqstorage]>=1.25.0
+google-cloud-bigquery-storage[fastavro]>=1.0.0
```

Comment:
I am not sure what should be done concerning this.
`fastavro` was needed for the most basic functionality for me, and i think `pyarrow` may also be needed for other basic scenarios.
Should we include both as extras?